keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+
+import tensorflow as tf
+
+from keras_cv.utils import preprocessing
+
+
+class RandomShear(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
+    """Randomly shears an image.
+
+    Args:
+        x: float, 2 element tuple, or `None`.  For each augmented image a value is
```

Comment:
Is this value based on PI? eg if I want the image to shear between 0 and 60 degree, should I put (0, 60/PI) here? (I think its better to have some sample value/range here)

Reply:
Updated with value information - it is a percentage of the image.  Thanks